pages/publish-nft-book/index.vue

```diff
@@ -28,17 +28,20 @@
               @submit="handleUploadSubmit"
             />
           </div>
-        </div>
+          <div v-else-if="step === 1">
+            <RegisterISCN ref="registerISCN" @submit="handleIscnSubmit" />
+          </div>
 
-        <!-- Navigation Buttons -->
-        <div class="flex gap-2 justify-center mt-4">
-          <UButton
-            v-if="hasFiles"
-            :disabled="step === steps.length - 1 || shouldDisableNext"
-            @click="nextStep"
-          >
-            {{ nextText }}
-          </UButton>
+          <!-- Navigation Buttons -->
+          <div class="flex gap-2 justify-center mt-4">
+            <UButton
+              v-if="step > 0 || step === 0 && hasFiles"
```

Comment:
```suggestion
              v-if="(step > 0 || step === 0) && hasFiles"
```
Please add `()`, not sure if this suggestion is what expected

Reply:
[🎨 Improve nextStep logic](https://github.com/likecoin/nft-book-press/pull/292/commits/0af8854eb74fd97ade60972d8ee76ee1267ffa2d)